packages/odyssey-react/src/components/Box/Box.tsx

```diff
@@ -316,7 +316,7 @@ export interface BoxProps {
    * Whether to apply the base font size class
    * @default base
    */
-  fontSize?: "base" | "inherit" | false;
+  fontSize?: "base" | "caption" | "inherit" | false;
```

Comment:
can we reuse the styles for these as they exist in `Text` already? Your component code would then look like this:

```
<Box ... />
  <Text color="danger" fontSize="caption">Hello World</Text>
</Box>
```
or whatever composition is necessary

Reply:
This has been removed.